kitchen-sink/src/views/MainPanel.js

```diff
@@ -27,9 +27,9 @@ const MainPanel = kind({
 
 	render: (props) => (
 		<Panel {...props}>
-			<Divider spacing="normal" startSection>
+			<Heading spacing="medium" showLine>
```

Comment:
I don't think we need `spacing="medium"` here since medium spacing is automatically applied for medium Headings, and Heading defaults to "medium".